components/elm/src/biogeophys/WaterStateType.F90

```diff
@@ -35,7 +35,8 @@ module WaterstateType
      real(r8), pointer :: bw_col                 (:,:) ! col partial density of water in the snow pack (ice + liquid) [kg/m3] 
      real(r8), pointer :: finundated_col         (:)   ! fraction of column that is inundated, this is for bgc caclulation in betr
      real(r8), pointer :: h2osoi_tend_tsl_col    (:)   ! col moisture tendency due to vertical movement at topmost layer (m3/m3/s)
- 
+     real(r8), pointer :: excess_ice             (:,:) ! excess ground ice in polygonal tundra areas [kg/m2]
```

Comment:
WaterStateType is deprecated, so this should be moved to `column_water_state`  type instead

Reply:
Good catch - this variable is declared but never used. There is another variable named excess_ice that is declared in column_water_state, and that is the one used by the rest of the new code for this PR. I will remove the extraneous declaration from WaterStateType.F90.